src/Symfony/Component/Serializer/Tests/Normalizer/AbstractNormalizerTest.php

```diff
@@ -103,4 +104,20 @@ public function testObjectToPopulateWithProxy()
 
         $this->assertSame('bar', $proxyDummy->getFoo());
     }
+
+    public function testObjectWithStaticConstructor()
+    {
+        $normalizer = $this->getMockBuilder(ObjectNormalizer::class)
```

Comment:
I don't understand why we would need too use a mock here. Why not using the `ObjectNormalizer` directly?